content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,35 +1,45 @@
-=======================
-Variants BOM Management
-=======================
-
-Odoo allows you to use one bill of materials for multiple variants of
-the same product. To activate variants, simply go to :menuselection:`Configuration --> Settings
---> Variants`.
-
-.. image:: product_variants/bom-variants1.png
-    :align: center
-
-You will then be able to specify which BOM component lines are to be used in
-the manufacture of each product variant. You may specify multiple
-variants for each line. If no variant is specified on a component line,
-then this component will be used in the BOM of all variants. 
-
-When defining variant BoMs on a line-item-basis, the **Product
-Variant** field in the main section of the BoM should be left blank as shown 
-below. This field is used when creating a BoM for one specific variant
-of a product only.
-
-
-.. image:: product_variants/kit-bom1.png
-    :align: center
-
-Note that the same principle applies for the configuration 
-of operations.
-
-.. image:: product_variants/kit-bom2.png
-    :align: center
-
-And for the production of by-products. 
-
-.. image:: product_variants/kit-bom3.png
-    :align: center
\ No newline at end of file
+==================================
+Managing BoMs for product variants
+==================================
+
+Odoo allows one bill of materials (BoM) to be used for multiple variants of the same product.
+Having a consolidated BoM for a product with variants saves time by preventing the need to manage
+multiple BoMs.
+
+Activate product variants
+=========================
+
+To activate variants, simply navigate to :menuselection:`Inventory --> Configuration --> Settings
+--> Products`, and then enable the :guilabel:`Variants` option. After that, click :guilabel:`Save`
+to apply the setting. For more information on configuring product variants, refer to :doc:`this
+page <../../../sales/sales/products_prices/products/variants>`.
```

Comment:
Tip: If the relative path forces you to go as far back as the root of the `applications` directory, you can eventually craft your paths like this: `/applications/sales/sales/products_prices/products/variants`.